macosx-shell-scripts.plugin.zsh

```diff
@@ -0,0 +1,2 @@
+#!/bin/zsh
+path+=("${0:h}")
```

Comment:
This file is loaded automagically via any ZSH framework upon instantiation. ZSH treats path as an array view on your standard PATH variable, so this just appends PATH with a single element. $0 is of course the script name, and the modifier `:h` is equivalent to `dirname`